packages/fetch/src/index.ts

```diff
@@ -0,0 +1,162 @@
+import {
+  camel,
+  ClientBuilder,
+  ClientDependenciesBuilder,
+  ClientGeneratorsBuilder,
+  generateFormDataAndUrlEncodedFunction,
+  generateVerbImports,
+  GeneratorDependency,
+  GeneratorOptions,
+  GeneratorVerbOptions,
+  GetterPropType,
+  stringify,
+  toObjectString,
+  generateBodyOptions,
+  isObject,
+} from '@orval/core';
+
+const PARAMS_SERIALIZER_DEPENDENCIES: GeneratorDependency[] = [
+  {
+    exports: [
+      {
+        name: 'qs',
```

Comment:
Do we really need qs? I would prefer not having a dependencies to add for that personally. What do you think?

Reply:
That's right. I left it there with the assumption that it would be used in later development, but I want to minimize dependencies on `fetch`, so I agree with the design without it.

Reply:
Fixed by https://github.com/anymaniax/orval/pull/1353/commits/805fbde24c78bc43903d604674aee5a718e199dc